plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -250,7 +279,7 @@ fu_uefi_dbx_device_init(FuUefiDbxDevice *self)
 	fu_device_set_name(FU_DEVICE(self), "UEFI dbx");
 	fu_device_set_summary(FU_DEVICE(self), "UEFI revocation database");
 	fu_device_build_vendor_id(FU_DEVICE(self), "UEFI", "Linux Foundation");
```

Comment:
Likewise this isn't really LF - isn't this Microsoft?

Reply:
Maybe in the "common" case the vendor should be MS?

Reply:
I think this should be set from the quirk file, e.g. `UEFI\CRT_{sha256}` -> "Microsoft"

Reply:
But then it's going to mean quirk file needs to refresh every new dbx, right?  Maybe we should have metadata set it too for dbx file in case quirk file is out of date.

Reply:
Nope, just ever new KEK -- and thankfully those are few and far between.